aliyun/src/test/java/org/apache/iceberg/aliyun/oss/AliyunOSSExtension.java

```diff
@@ -39,18 +39,13 @@ default String testBucketName() {
   }
 
   @Override
-  default Statement apply(Statement base, Description description) {
-    return new Statement() {
-      @Override
-      public void evaluate() throws Throwable {
-        start();
-        try {
-          base.evaluate();
-        } finally {
-          stop();
-        }
-      }
-    };
+  default void afterAll(ExtensionContext context) throws Exception {
+    stop();
+  }
+
+  @Override
+  default void beforeAll(ExtensionContext context) throws Exception {
```

Comment:
I don't think we need the `default` for these two methods

Reply:
`AliyunOSSExtension` is an interface. If we want it to have a common method for it's implement, I guess we could only use such way?

Reply:
the `beforeAll` and `afterAll` methods are from the junit5 extension `BeforeCallback` and `AfterCallback` interface